core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/workflow/WorkflowCompiler.scala

```diff
@@ -157,7 +157,6 @@ class WorkflowCompiler(val workflowInfo: WorkflowInfo, val context: WorkflowCont
             Option.apply(op.getOutputSchema(inputSchemaMap(op).map(s => s.get).toArray))
           } catch {
             case e: Throwable =>
-              e.printStackTrace()
```

Comment:
why are we removing this?

Reply:
this is by mistake, should be reverted

Reply:
actually, it's better to replace it to log.error as this outputs to stderr directly.